zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/X509AuthenticationConfig.java

```diff
@@ -48,21 +48,30 @@ public static X509AuthenticationConfig getInstance() {
     return instance;
   }
 
+  // The following System Property keys are used to extract clientId from the client cert.
```

Comment:
Please strongly consider creating a separate PR for non-logical changes since this adds difficulty to reviewers in the future.

Reply:
Sorry about the inconvenience. I tried to add some comments for `SET_X509_CLIENT_ID_AS_ACL` field to clarify the behavior and realized that the comment is not using the correct format, so changed them. Will pay attention in the future.